packages/loader/container-loader/src/container.ts

```diff
@@ -157,6 +157,29 @@ export enum ConnectionState {
     Connected,
 }
 
+/** Invoke the callback once deltaManager has processed all ops known at the time this function is called */
+function waitForOps(container: IContainer, resolve: (hasCheckpointSequenceNumber: boolean) => void) {
```

Comment:
`waitForOps` is too generic. Maybe this should be called `waitToProcessKnownOps`?

Reply:
Or `catchUpToKnownOps`